src/hotspot/cpu/x86/stubGenerator_x86_64_arraycopy.cpp

```diff
@@ -2476,6 +2481,281 @@ address StubGenerator::generate_unsafe_copy(const char *name,
 }
 
 
+//  Generate 'unsafe' set memory stub
+//  Though just as safe as the other stubs, it takes an unscaled
+//  size_t argument instead of an element count.
+//
+//  Input:
+//    c_rarg0   - destination array address
+//    c_rarg1   - byte count (size_t)
+//    c_rarg2   - byte value
+//
+// Examines the alignment of the operands and dispatches
+// to an int, short, or byte fill loop.
+//
+address StubGenerator::generate_unsafe_setmemory(const char *name,
+                                                 address byte_fill_entry) {
```

Comment:
Need to add UnsafeSetMemoryMark on similar lines as UnsafeCopyMemoryMark to handle page error.

Reply:
Will do.